tests/src/FunctionalJavascript/ListPagesPresetFiltersTest.php

```diff
@@ -478,7 +483,7 @@ protected function setListPageFilters(NodeInterface $node, array $filters): void
    *   The expected count.
    */
   protected function assertResultCount(int $count): void {
-    $items = $this->getSession()->getPage()->findAll('css', '.node--view-mode-teaser');
+    $items = $this->getSession()->getPage()->findAll('css', 'li > article');
```

Comment:
Why we have changed CSS selector? Now selector here looks a bit less specific (as based only on tag names).